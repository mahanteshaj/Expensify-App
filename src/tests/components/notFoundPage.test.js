import React from 'react'
import  {shallow } from 'enzyme'

import NotFoundPage from '../../components/notFoundPage'
test('test should render not found page ',()=>{
    const wrapper =shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()
})