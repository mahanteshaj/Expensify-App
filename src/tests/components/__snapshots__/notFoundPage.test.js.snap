// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should render not found page  1`] = `
<div>
  Page Not found-
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
